docs/01-Introduction.md

```diff
@@ -3,3 +3,10 @@
 ## 1.1 What is LORIS?
 
 ## 1.2 How does LORIS work?
+
+
+### Deidentification of imaging datasets
+
+All DICOM datasets transferred to the DCC must be free of any identifying
```

Comment:
I would refrain from referring to DCC as some projects do not use DCC (CCNA is one such example):)
Also, the only thing LORIS prevents during the upload/insertion is having either PatientName or PatientID other than the PSCID_CandID_VL convention. Any other identifying information is not caught/is allowed.